pegasus/sites.v3/code.org/layouts/default.haml

```diff
@@ -1,4 +1,5 @@
 %link{:href=>"/shared/css/warning-banner.css", :rel=>"stylesheet"}
+%script{src: minifiable_asset_path("js/pegasusCommon.js")}
```

Comment:
Actually @breville there might be a more common place to put this [theme_common_header_after](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/views/theme_common_head_after.haml)?
I also don't understand the difference between /layouts and /themes in pegasus. Is it true that one of each is always used on a page? Right now I put all this js in layouts, but that might be wrong.